pipeline-braille-scripts/epub3-to-pef/src/main/resources/css/default.scss

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
It feels weird that the rules are defined here while the corresponding documentation is in xml-to-pef.